docs/man_pages/publishing/appmanager-upload-android.md

```diff
@@ -1,28 +1,32 @@
 appmanager upload android
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
 General | `$ appbuilder appmanager upload android [--certificate <Certificate ID>] [--download]`   
 
-Builds the project and uploads the application to Telerik AppManager.
-After the upload completes, you need to go to your app in Telerik AppManager, manually configure it for distribution and publish it.
-<% if (isMobileWebsite) {%>
-This command is not applicable to Mobile Website projects.
+Builds the project for Android and uploads the application to Telerik AppManager. <% if(isHtml) { %>After the upload completes, you need to go to your app in Telerik AppManager, manually configure it for distribution and publish it.<% } %> 
```

Comment:
Do you think it is a good idea to give a link to AppManager? i.e. instead of  uploads the application to Telerik AppManager to write  uploads the application to  <% if(isHtml) { %>(Telerik AppManager)[https://platform.telerik.com/appmanager]<% } else { %>Telerik AppManager<% } %>